### ISSUE TITLE
1.10.4 version is empty/broken

### ISSUE THREAD

**Issue by yogeshgadge** [US-CBP/cbp-theme#276]
Current version as of this issue is 1.10.4
```bash
npm install cbp-theme
ls -l node_modules/cbp-theme/
total 20K
-rw-r--r-- 1 DGH6TV6 1049089 1.3K Oct 26  1985 LICENSE.MD
-rw-r--r-- 1 DGH6TV6 1049089 4.7K Apr 26 16:46 package.json
-rw-r--r-- 1 DGH6TV6 1049089 6.4K Oct 26  1985 README.md


```

**Comment by davezen1** [comment]
It should be fixed with 1.10.5 version. Let us know if you still have issues.

cc @HIGGO36

**Comment by yogeshgadge** [comment]
Fixed in 1.10.5
